README.md

```diff
@@ -68,6 +68,25 @@ must be applied via [patches](patches). See [below](#patches) for more info on p
 1. After the PR is merged to update a component, coordination is often needed in the darc dependency flows. The source-build-external update
 may need to flow in at the same time as the cooresponding changes in product repos which take a dependency on the new component version.
 
+### Updating an External Component Used in a Toolset Repo and in a Target Repo
+
+When updating a component that is used in a repository which is built before source-build-externals during the product build (toolset repos) and is used in a repo that is built after source-build-externals (target repos), please adhere to the following steps:
+
+> [!NOTE]
+>
+> Current toolset repositories are:
+> - [Arcade](https://github.com/dotnet/arcade)
+> - [Source-build-reference-packages](https://github.com/dotnet/source-build-reference-packages/)
+> - [Cecil](https://github.com/dotnet/cecil)
+> - [Command-line-api](https://github.com/dotnet/command-line-api/)
+> - [Emsdk](https://github.com/dotnet/emsdk)
+
+1. **Add the updated component**: Include the component with the updated version as a new submodule in source-build-externals named `<component>-<new-version>` (this new submodule should be separate from any previously existing version's submodule). Rename the component's old submodule to `<component>-<old-version>`. Also rename the old component's patches directory in `patches/` (if it exists) and the project file in `repo-projects/` to match the submodule name of `<component>-<old-version>`.
```

Comment:
Would it be valuable to reference a pattern to copy?  A link to a PR which did this?